extension.js

```diff
@@ -45,6 +45,7 @@ function enable() {
                 module.enable();
             });
         });
+        return GLib.SOURCE_REMOVE;
```

Comment:
This is somewhat unrelated to this PR, sorry about that.
While reading the gnome shell source, I found that GLib `timeout_add` and `idle_add` functions expect a flag as return value to determine if it should be called again. This is not mandatory since no return statement in js is falsy but it's cleaner that way and mirrors the gnome-shell style (https://gitlab.gnome.org/search?utf8=%E2%9C%93&search=SOURCE_REMOVE&group_id=&project_id=546&search_code=true&repository_ref=master)
See https://developer.gnome.org/glib/stable/glib-The-Main-Event-Loop.html#G-SOURCE-REMOVE:CAPS